bazel/scripts/test_python_service_imports.sh

```diff
@@ -85,7 +85,7 @@ do
 # the same options as the 'bazel run' below. For 'bazel run' the option is set
 # in the bazelrc, due to https://github.com/bazelbuild/bazel/issues/11920
     echo "Testing service: ${SERVICE}"
-    if timeout --preserve-status 5 bazel run "${SERVICE}" 2>&1  | tee "${LOGGING_FILE}";
+    if timeout --signal 9 --preserve-status 5 bazel run "${SERVICE}" 2>&1  | tee "${LOGGING_FILE}";
```

Comment:
Do you know why `TERM` is no longer enough?

Reply:
See in the description - before pipelined did not get started into a state where it was not possible to end it gracefully

Reply:
https://github.com/magma/magma/blob/ff9e3d39c8edee8e708c28fd5f5f82630316eee2/lte/gateway/python/magma/pipelined/service_manager.py#L612

Pipelined goes into a sleep loop if it cannot reach redis and ignores SIGTERMs. Maybe we should create a ticket to fix that, it will block magmad restarts too...